ui/src/components/Form.jsx

```diff
@@ -9,38 +9,52 @@ import ParcelSize from "./ParcelSize";
 import ParcelType from "./ParcelType";
 import Summary from "./Summary";
 import useMessage from "../hooks/messages";
+import { PACKAGE_SIZES } from "../config";
 import PaymentSuccess from "./PaymentSuccess";
 
 function Form() {
   const { displayError } = useMessage();
-
-  var boxSizes = [
-    { id: 1, name: "Small Box", width: 35, height: 16, length: 45, weight: 2 },
-    { id: 2, name: "Medium Box", width: 46, height: 46, length: 64, weight: 20 },
-    { id: 3, name: "Large Box", width: 150, height: 150, length: 150, weight: 30 },
-  ];
-
-  var letterSizes = [
-    { id: 4, name: "Small Letter", weight: 100, length: 24, width: 16 },
-    { id: 5, name: "Large Letter", weight: 750, length: 35, width: 25 },
-  ];
-
   const [selectedPackageType, setSelectedPackageType] = useState(undefined);
   const [selectedBoxSize, setSelectedBoxSize] = useState(undefined);
   const [selectedDocumentSize, setSelectedDocumentSize] = useState(undefined);
   const [selectedPaymentType, setSelectedPaymentType] = useState(undefined);
   const [currentPage, setCurrentPage] = useState(0);
   const [progress, setProgress] = useState(25);
   const [formData, setFormData] = useState(undefined);
+  const [packageSizes, setPackageSizes] = useState(undefined);
+
+  useEffect(() => {
+    const requestOptions = {
+      method: "GET",
+      headers: { "Content-Type": "application/json" },
+    };
+    (async function () {
+      try {
+        const response = await fetch(PACKAGE_SIZES, requestOptions);
```

Comment:
We should be fetching PACKAGE_OPTIONS here. We won't be able to calculate price otherwise.